lib/Cron/Daemon.php

```diff
@@ -51,8 +51,9 @@ private function getWorkers() {
     yield $this->workers_factory->createPremiumKeyCheckWorker($this->timer);
     yield $this->workers_factory->createBounceWorker($this->timer);
     yield $this->workers_factory->createExportFilesCleanupWorker($this->timer);
+    yield $this->workers_factory->createBeamerkWorker($this->timer);
     yield $this->workers_factory->createInactiveSubscribersWorker($this->timer);
     yield $this->workers_factory->createWooCommerceSyncWorker($this->timer);
-    yield $this->workers_factory->createAuthorizedSendingEmailsCheckWorker($this->timer);
+    yield $this->workers_factory-> createAuthorizedSendingEmailsCheckWorker($this->timer);
```

Comment:
Why did you modify this line?

Reply:
That's my VSCode adding spaces when I paste something :(